feathr_project/test/samples/test_notebooks.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+try:
+    import papermill as pm
+    import scrapbook as sb
+except ImportError:
+    pass  # disable error while collecting tests for non-notebook environments
+
+
+SAMPLES_DIR = (
+    Path(__file__)
+    .parent     # .../samples
+    .parent     # .../test
+    .parent     # .../feathr_project
+    .parent     # .../feathr (root of the repo)
+    .joinpath("docs", "samples")
+)
+NOTEBOOK_PATHS = {
+    "nyc_taxi_demo": str(SAMPLES_DIR.joinpath("nyc_taxi_demo.ipynb")),
+}
+
+
+@pytest.mark.notebooks
```

Comment:
After adding this, sounds like it's possible to run tests on the notebooks?

Reply:
With a custom marker, you can control which tests you want to run or not.
E.g. I used a custom marker `notebooks`. To run only the tests marked by this marker,

`pytest -m notebooks` 

Or to exclude,
`pytest -m "not notebooks"`